bpf/bpf_lxc.c

```diff
@@ -53,6 +53,32 @@ static __always_inline bool redirect_to_proxy(int verdict, __u8 dir)
 }
 #endif
 
+#ifdef ENABLE_CUSTOM_CALLS
+/* Encode return value and identity into cb buffer. This is used before
+ * executing tail calls to custom programs. "ret" is the return value supposed
+ * to be returned to the kernel, needed by the callee to preserve the datapath
+ * logics. The "identity" reflects the local endpoint: the source of the packet
+ * on ingress path, or its destination on the egress path. We encode it so that
```

Comment:
nit, on first read I found it ambiguous whether the "reflects" + "source of the packet" were trying to imply that the "identity" parameter is really an endpoint ID or a security identity that is common across multiple endpoints. A couple of extra words help to dispell that:

```suggestion
 * logics. The "identity" is the security identity of the local endpoint: the source of the packet
 * on ingress path, or its destination on the egress path. We encode it so that
```